nav2_tasks/include/nav2_tasks/rate_controller_node.hpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+  
+#ifndef NAV2_TASKS__RATE_CONTROLLER_NODE_HPP_
+#define NAV2_TASKS__RATE_CONTROLLER_NODE_HPP_
+
+#include <chrono>
+#include "behavior_tree_core/decorator_node.h"
+
+namespace nav2_tasks
+{
+
```

Comment:
A short description of what a RateController does.

Reply:
@mhpanah Yes, I can investigate that. We could use a multiple of the tick count if that was available to the nodes. Can consider for a subsequent addition.

Reply:
@orduno I think RateController is pretty clear. It's controlling the rate at which the child is invoked.